client/lib/post-normalizer/rule-wait-for-images-to-load.js

```diff
@@ -58,6 +58,15 @@ export default function waitForImagesToLoad( post ) {
 				return find( post.images, { src: image.src } );
 			} ), Boolean );
 
+			// this adds naturalHeight and naturalWidth to the images
```

Comment:
that line still works, the only issue is the comment that needs to be modified 👍